### ISSUE TITLE
Blue Solid Light not connecting to network

### ISSUE THREAD

**Issue by droidforever** [ct-Open-Source/tuya-convert#210]

**Comment by droidforever** [comment]
I flashed this per the guide but now the wifi light is solid blue and won't reset.  I logged into the tasmota page and set my wifi SSID and my password but it doesn't show up  in my devices.  Is there a way to force this plug back into flash mode so I can attempt it again or is it just bricked? Tuya Smart Plug 10AL250V I am sorry I have no idea what firmware this unit came with.

**Comment by droidforever** [comment]
I flashed this per the guide but now the wifi light is solid blue and won't reset. I logged into the tasmota page and set my wifi SSID and my password but it doesn't show up in my devices. Is there a way to force this plug back into flash mode so I can attempt it again or is it just bricked? Tuya Smart Plug 10AL250V I am sorry I have no idea what firmware this unit came with. Blue Solid Light not connecting to network
[logfiles.zip](https://github.com/ct-Open-Source/tuya-convert/files/3259848/logfiles.zip)
